alvr/graphics/resources/stream.wgsl

```diff
@@ -169,10 +169,21 @@ fn chroma_key_mask(color: vec3f) -> f32 {
     let end_min = vec3f(pc.ck_channel0.z, pc.ck_channel1.z, pc.ck_channel2.z);
     let end_max = vec3f(pc.ck_channel0.w, pc.ck_channel1.w, pc.ck_channel2.w);
 
-    let start_mask = smoothstep(start_min, start_max, color);
-    let end_mask = smoothstep(end_min, end_max, color);
+    let start_value_f2 = smoothstep(start_min.yz, start_max.yz, color.yz);
+    let end_value_f2 = smoothstep(end_min.yz, end_max.yz, color.yz);
+    let sv_value_f2 = max(start_value_f2, end_value_f2);
+    let sv_value = max(sv_value_f2.x, sv_value_f2.y);
```

Comment:
Don't use the word "value" here because it has a different meaning with HSV. also don't postfix (or prefix) with the type, that's not a good coding practice in modern languages. I would keep the word "mask" instead. I would use in order: start_mask, end_mask, sv_mask. And the old sv_value I would move in the last return without variable binding.